docker-app/qfieldcloud/core/admin.py

```diff
@@ -1079,10 +1079,7 @@ class OrganizationAdmin(QFieldCloudModelAdmin):
 
     @admin.display(description=_("Active users (last billing period)"))
     def active_users(self, instance) -> int:
-        try:
-            return instance.current_subscription_vw.active_users_count
-        except Exception:
-            return None
+        return instance.current_subscription_vw.active_users_count
```

Comment:
@why-not-try-calmer do you remember why you had the `except Exception` here?
I changed it because `None` doesn't fit the return type `int` which might cause exceptions in other places, so I think in case of troubles it's better to have the root cause exception here (and handle specific exceptions if needed)

Reply:
I remember understanding from https://github.com/opengisch/qfieldcloud/pull/591#discussion_r1146320252 that there was a potential exception-raiser upstream (I think it wasn't safe to assume that every organization had a current active subscription). But now looking at the code, `public/docker-app/qfieldcloud/core/models.py:Subscription.get_or_create_current_subscription(self)` suggests that the assumption is safe indeed, so your change seems justified.

Reply:
Thanks for the link, that comment suggests it's safe to return `None` which means we should adapt the return type to `Optional` instead.
Should we change the caught exception to `AttributeError` (most of the time I prefer specific exceptions, generic ones often risk masking other issues)? Or is there something else to do (since you are surprised)?

Reply:
See commit below for suggestion.